src/index.html

```diff
@@ -7,8 +7,67 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@500&family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#">
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="logo"
+        >
+      </a>
+      <nav class="nav__bar">
+        <ul class="nav__bar-list">
+          <li class="nav__bar-item">
+            <a href="#" class="nav__bar-link is-active">
+              Apple
+            </a>
+          </li>
+          <li class="nav__bar-item">
+            <a href="#" class="nav__bar-link">
+              Samsung
+            </a>
+          </li>
+          <li class="nav__bar-item">
+            <a href="#" class="nav__bar-link">
+              Smartphones
+            </a>
+          </li>
```

Comment:
as those are on the same level of nesting and there are quite a lot of them - you can add some empty lines between them for better readability
```suggestion
        <ul class="nav__bar-list">
          <li class="nav__bar-item">
            <a href="#" class="nav__bar-link is-active">
              Apple
            </a>
          </li>
          
          <li class="nav__bar-item">
            <a href="#" class="nav__bar-link">
              Samsung
            </a>
          </li>
          
          <li class="nav__bar-item">
            <a href="#" class="nav__bar-link">
              Smartphones
            </a>
          </li>
```